frontend/web/components/ErrorMessage.js

```diff
@@ -34,6 +34,15 @@ export default class ErrorMessage extends PureComponent {
                 .join('<br/>'),
             }}
           />
+        ) : this.props.exceeded ? (
```

Comment:
We shouldn't pollute generic components.

Reply:
New component Created